### PR TITLE
feat: add ref to different jina products

### PR DESCRIPTION
**Pull request by FionnD**
### All Submissions:

* [X ] Have you followed the [documentation style guide](https://github.com/jina-ai/docs/tree/master/page_templates/style_guide.md)? 

Content: 
We don't explain to a user what's what.

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -19,6 +19,17 @@ Each of the four Jina products has the following documentation types.
 
 ------------
 
+Jina Products
+**********************
+
+
+- **Jina Core**: is the open-source neural search framework, it allows you to build search systems for any kind of data.
+- **Jina Dashboard**: is a low-code monitoring and management environment for Jina. With the Dashboard you can build Jina Flows, manage them from a central location and get insights into the health of the Flows.
+- **Jina Hub**: is a centralized registry, where you can share and discover custom and community driven Jina Pods and Apps tailored to specific use cases of neural search.
```
Comment (aga11313) [inline index.rst]: we dont offer Apps yet so maybe take that out

**Review by aga11313 — APPROVED** [review]

**Review by bwanglzu — APPROVED** [review]
LGTM!

File: index.rst (hunk 1 above)
Comment (FionnD) [reply index.rst]: Are Apps something different? I would assume they are just pods.

Comment (FionnD) [reply index.rst]: But yeah I can remove it

Comment (aga11313) [reply index.rst]: its something else we have plans for in the future i think

**Pull request merged**